canonical-web-and-design/www.ubuntu.com
# Add ai/consulting page

## Done

Add ai/consulting page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/ai/consulting](http://0.0.0.0:8001/ai/consulting)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Got to the page and make sure it complies with the [copy doc](https://docs.google.com/document/d/19Mx1zRJKqH2hz5kRF_oDsgyFUw67FgJvW4I_OGda-wk/edit#heading=h.qo896jpcbdg0)


## Issue / Card

Fixes #5083
